mycore-pi/src/main/java/org/mycore/pi/condition/MCRPIXPathPredicate.java

```diff
@@ -0,0 +1,24 @@
+package org.mycore.pi.condition;
+
+import org.jdom2.filter.Filters;
+import org.jdom2.xpath.XPathExpression;
+import org.jdom2.xpath.XPathFactory;
+import org.mycore.common.MCRConstants;
+import org.mycore.datamodel.metadata.MCRBase;
+
+public class MCRPIXPathPredicate extends MCRPIPredicateBase
+    implements MCRPICreationPredicate, MCRPIObjectRegistrationPredicate {
+    public MCRPIXPathPredicate(String propertyPrefix) {
+        super(propertyPrefix);
+        final String xPath = requireProperty("XPath");
+        XPathFactory factory = XPathFactory.instance();
+        expr = factory.compile(xPath, Filters.fpassthrough(), null, MCRConstants.getStandardNamespaces());
+    }
+
+    final private XPathExpression<Object> expr;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5472577)